### PR TITLE
Add doc for start singleton mode

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: content/en/getting-started/installation/register-a-cluster.md [hunk 1]
```diff
@@ -119,6 +119,41 @@ to the hosting cluster to register the managed cluster to the hub.
...
+
+{{< tabs name="clusteradm join" >}}{{% tab name="kind" %}}
+  ```shell
+  # NOTE: For KinD clusters use the parameter: --force-internal-endpoint-lookup
+  clusteradm join \
+      --hub-token <your token data> \
+      --hub-apiserver <your hub cluster endpoint> \
+      --wait \
+      --cluster-name "cluster1" \    # Or other arbitrary unique name
+      --mode singleton \
```
Comment (zhujian7) [inline content/en/getting-started/installation/register-a-cluster.md:139]: I tried the clusteradm in my lcoal environment, the help info shows:
```
    -m, --mode='default':
        mode to deploy klusterlet, can be default or hosted

    --singleton=false:
        If true, deploy singleton mode of klusterlet to have registration and work agents run in a single pod. This is
        an alpha stage flag.
```

so please confirm if `--mode singleton` is right

Comment (qiujian16) [reply content/en/getting-started/installation/register-a-cluster.md:139]: oops, good catch, it should be --singleton

**Pull request merged**